pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -739,6 +739,10 @@ def test_eq(self):
 
         assert DateOffset(milliseconds=3) != DateOffset(milliseconds=7)
 
+    def test_milliseconds_combination(self):
+        # 10525
+        DateOffset(days=1, milliseconds=1)
```

Comment:
Can you add an assertion that the object has been properly constructed?

Reply:
TBH, I prefer to restore the previous comment (just not as docstring). Similarly to this test:

https://github.com/pandas-dev/pandas/blob/1788491051f25c3ba82c517e0ceedfcbf0b9c621/pandas/tests/tseries/offsets/test_offsets.py#L912-L926

The whole point is to assert that the construction of the object does not fail (anymore), which is the whole point of the issue...

Is that ok?

Reply:
fixed